### DISCUSSION TITLE
Rabbimq cluster support

### DISCUSSION THREAD

**Discussion by CuriousAnkush** [ruby-amqp/bunny#620] [Q&A] [answered]
Does bunny support connecting to a Rabbimq cluster instead of an individual node.

**Comment by michaelklishin** [comment, marked as answer]
The two are not mutually exclusive, as documented in [Clustering and Clients](https://www.rabbitmq.com/docs/clustering#clustering-and-clients).

With AMQP 0-9-1 any client can connect to any node, the node will route the traffic as needed.
